apis/config/v1beta2/types.go

```diff
@@ -34,6 +34,23 @@ type CoschedulingArgs struct {
 	DeniedPGExpirationTimeSeconds *int64 `json:"deniedPGExpirationTimeSeconds,omitempty"`
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// LoadVariationRiskBalancingArgs holds arguments used to configure LoadVariationRiskBalancing plugin.
```

Comment:
Can you revisit this file to just introduce minimum changes? (i.e., don't reorder the code blocks which is hard to track the diff).

(Applies to some other files like v1beta3/types.go)